workspaces/frontend/src/app/pages/Workspaces/Details/WorkspaceDetailsOverview.tsx

```diff
@@ -29,9 +30,11 @@ export const WorkspaceDetailsOverview: React.FunctionComponent<WorkspaceDetailsO
     <DescriptionListGroup>
       <DescriptionListTerm>Labels</DescriptionListTerm>
       <DescriptionListDescription>
-        {Object.entries(workspace.podTemplate.podMetadata.labels)
-          .map(([key, value]) => `${key}=${value}`)
-          .join(', ')}
+        {Object.entries(workspace.podTemplate.podMetadata.labels).map(([key, value]) => (
+          <Label key={key} style={{ margin: '0.10rem' }}>
```

Comment:
I'm wondering if we should use the compact version of the Label component
https://www.patternfly.org/components/label/#compact-labels
```suggestion
          <Label key={key} style={{ margin: '0.10rem' }} isCompact>
```